.github/workflows/update-kernels.yml

```diff
@@ -21,13 +44,31 @@ jobs:
         with:
           ref: kernels
 
+      - name: Setup jq
+        run: apt-get update && apt-get install jq -y
+
       - name: Run crawler for x86_64
         run: |
-          kernel-crawler crawl --distro=* > x86_64/list.json
+          mkdir x86_64_tmp
```

Comment:
I'd use `$RUNNER_TEMP/x86_64` so that the folder is outside of the repo! Same for aarch64!